operator_ui/src/pages/Jobs/RegionalNav.tsx

```diff
@@ -232,7 +227,7 @@ const RegionalNavComponent = ({
                   color="secondary"
                   className={classes.jobSpecId}
                 >
-                  {jobSpecId}
+                  {(job && job.attributes.name) || jobSpecId}
```

Comment:
Can you use nullish operator `??`

Reply:
+ a few other places

Reply:
I can't use nullish because the name can be an empty string and nullish only checks for undefined or null.